Source/WebCore/style/PropertyCascade.h

```diff
@@ -124,29 +124,29 @@ class PropertyCascade {
     // The CSSPropertyID enum is sorted like this:
     // 1. CSSPropertyInvalid and CSSPropertyCustom.
     // 2. Normal longhand properties (high priority ones followed by low priority ones).
-    // 3. Deferred longhand properties.
+    // 3. Logical longhand properties.
```

Comment:
Ditto

Reply:
This comment is bit weird too. PropertyCascade only deals with shorthands and does not care about longhands at all.